pkg/download/jsoncreator/json_creator_util.go

```diff
@@ -0,0 +1,15 @@
+package jsoncreator
```

Comment:
I guess this file should also be tagged with `unit` and `integration`

Reply:
I've tried adding tags before the package definition but then vscode automatic test detection fails to detect anything that doesn't start with package in the first line. Not sure how are you running the test locally, but I find really useful the visual studio code intelisense for golang to speed development. I might have to google more to find a workaround for vscode or github tags

Reply:
I guess this is caused by golang-lsp ignoring all tagged files by default.

Here is the issue for it. It also contains info how to specify tags it should use. 
https://github.com/golang/go/issues/29202